revaturelabs/ProjectForce
# Development

Fixes for save modal. To test:

- select a bar on the chart

- select the checkbox for Completed and checkbox for Review Completed

- click save

- check the Training__c you updated and its related project, make sure that the Review Completed checkbox is checked on the Training__c and the Completed checkbox is checked on the Project__c

- there may be a few new lines of apex code to test in ModalController
